src/ui/map.js

```diff
@@ -1393,7 +1393,8 @@ class Map extends Camera {
     }
 
     /**
-     * Returns a Boolean indicating whether the source is loaded.
+     * Returns a Boolean indicating whether the source is loaded. True if the source with
```

Comment:
should we format that True as `true`?

Reply:
There appears to be a bit of inconsistency in the API reference -- sometimes boolean values are formatted as `true` and other times as "true". I prefer the `true` format as well, but went with "True" due to capitalization. 

I am open to changing it/hearing with others think!

Reply:
I think we should standardize on `true` as it's a value within Javascript. In this particular case, we can sidestep the capitalization worries by rewording slightly. How about something like "Returns `true` if the source is loaded, otherwise `false`"?

Reply:
That makes sense @ryanhamley -- I made a small update. It's slightly wordier than your suggestion so as to still address the initial feedback motivating this PR (i.e. what does it mean for a source to be loaded). This means there are now two sentences starting with "Returns...", but the clarity might be worth the slight redundancy here.